django/core/checks/registry.py

```diff
@@ -17,6 +17,7 @@ class Tags(object):
     models = 'models'
     security = 'security'
     signals = 'signals'
+    staticfiles = 'staticfiles'
```

Comment:
Perhaps admin set a bad precedent -- it doesn't seem like contrib tags should have a reference "core" here.